docs/internationalisation/localisation.md

```diff
@@ -4,8 +4,8 @@ title: Localisation
 parent: Internationalisation
 ---
 
-Our platform has built in localisation so Creators can use the platform in their native language. 
+Our platform has built-in localisation so Creators can use the platform in their native language. 
 
-However, we do not currently offer localised naming for Theme nomenclature, like block names, or block variable names. Easol themes are built in a single language, currently all English.
+However, we do not currently offer localised naming for Theme nomenclature, like block names, or block variable names. Easol themes are  a single language, currently all English.
```

Comment:
**sugestion:** Grammarly's change here makes it a little unclear; I think the current version reads better.

Reply:
Completely agree, thanks for flagging